tests/e2e/utils/helpers.go

```diff
@@ -94,6 +94,44 @@ func SubnetConfigExists(subnetName string) (bool, error) {
 	return gen && sc, nil
 }
 
+func UpdateSubnetConfig(subnetName string, network models.Network, subnetID string) error {
```

Comment:
Could you rename this to `AddSubnetIDToSidecar` or something more specific. The name is a little misleading and vague.

Reply:
done